### PR TITLE
USHIFT-2463: Add PreRunModifier flag to skip RF

### PR DESCRIPTION
**Pull request by agullon**
Closes Jira [USHIFT-2463](https://issues.redhat.com/browse/USHIFT-2463)

Add `--prerunmodifier` [flag](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#programmatic-modification-of-test-data) to RF execution in `run.sh` to skip tests by name.

Example:
`./test/run.sh -o /Users/agullon/workspace/microshift/_output/_output -i /Users/agullon/workspace/microshift/_output/rf_variables.yaml -k 'CLI Output,ConfigMap Matches CLI' -v /Users/agullon/workspace/microshift/_output/venv suites/standard1/version.robot`
```
+ /Users/agullon/workspace/microshift/_output/venv/bin/robot --randomize all --prerunmodifier 'resources/SkipTests.py:CLI Output,ConfigMap Matches CLI' --loglevel TRACE -V /Users/agullon/workspace/microshift/_output/rf_variables.yaml -x junit.xml --outputdir /Users/agullon/workspace/microshift/_output/_output suites/standard1/version.robot
List of tests to be skipped:
 - CLI Output
 - ConfigMap Matches CLI
==============================================================================
Version :: Tests related to the version of MicroShift
==============================================================================
ConfigMap Contents :: Check the version of the server                 | PASS |
------------------------------------------------------------------------------
CLI Output :: Check the version reported by the process               | SKIP |
Test skipped using 'robot:skip' tag.
------------------------------------------------------------------------------
Metadata File Contents :: Ensure the metadata file contents match ... | PASS |
------------------------------------------------------------------------------
ConfigMap Matches CLI :: Ensure the ConfigMap is being updated bas... | SKIP |
Test skipped using 'robot:skip' tag.
------------------------------------------------------------------------------
Version :: Tests related to the version of MicroShift                 | PASS |
4 tests, 2 passed, 0 failed, 2 skipped
==============================================================================
Output:  /Users/agullon/workspace/microshift/_output/_output/output.xml
XUnit:   /Users/agullon/workspace/microshift/_output/_output/junit.xml
Log:     /Users/agullon/workspace/microshift/_output/_output/log.html
Report:  /Users/agullon/workspace/microshift/_output/_output/report.html
+ '[' '' ']'
```

### CODE REVIEW

File: test/run.sh [hunk 1]
```diff
@@ -24,10 +24,11 @@ Options:
   -v DIR             The venv directory. (${RF_VENV})
   -i PATH            The variables file. (${RF_VARIABLES})
   -s NAME=VALUE      To enable an stress condition.
+  -k SKIP_TESTS      List of Tests to skip.
```
Comment (ggiguash) [inline test/run.sh:27]: Nit: we should probably say: Comma separated list of tests to skip

File: test/run.sh [hunk 2]
```diff
@@ -101,6 +105,7 @@ else
     # shellcheck disable=SC2086
     "${RF_BINARY}" \
         --randomize all \
+        --prerunmodifier resources/SkipTests.py:${SKIP_TESTS:-} \
```
Comment (ggiguash) [inline test/run.sh:108]: Does this work if you do not run from the `microshift/tests` current directory?
We should probably specify a full path to the script, i.e. `${SCRIPTDIR}/resources/SkipTests.py`

File: test/run.sh (hunk 1 above)
Comment (ggiguash) [reply test/run.sh:27]: Ah, and while you're there, please also fix the above comment.
`an stress` -> `a stress`

File: test/run.sh (hunk 2 above)
Comment (agullon) [reply test/run.sh]: It does work, new commit pushed to set the full path.

**Pull request merged**